Site/CuratedContentEditorialWorkflow.md

```diff
@@ -0,0 +1,54 @@
+
+
+# A Proposed Editorial Workflow for BSSw.io Curated Content
+
+*Note: might be equally appropriate for other types of content, but we’re starting with Curted Content.*
+
+This is a rough outline of a process that I think might be suitable to our needs that can be
+carried out using tools provided by GitHub and perhaps some helper automations/bots/scripts.
+There is plenty of room for discussion and refinement.
+
+Reference: 
+
+[https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/projects/3](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/projects/3) 
+
+# Actors
+* **Anyone** who can submit a GitHub issue can use this approach to suggest a curated content topic
+  * Do we need to do anything to accommodate those who can’t (or don’t want to) submit a GH issue?
+* **Editorial Board Member**
+* **Author** who can fork the repository and submit a pull request for a new article
+  * Do we need to do anything to accommodate those who can’t (or don’t want to) use the GH pull request process?
+* **Editorial Assistant** is a role that was first recognized in the blog workflow.
+  It is someone familiar with the BSSw.io site and its operations, but who does not
+  need to have the technical expertise to evaluate content that an editorial board
+  member or subject matter expert would be expected to have.
+
+# Workflow
+1. **Anyone** creates a GitHub issue for a proposed curated content (CC) item
+   * We should use issue templates to tailor the requested data and automatically
+   apply an appropriate label/project board assignment
+1. The issue is added to the “Curated Content Ideas” (CCI) project board *Idea Backlog* column.
+   * Ideally, this should be done automatically on creation of the issue.  From research on the web,
+     it doesn’t appear that issue templates allow direct assignment to boards yet.  Maybe this can be
+     done with a bot for issues with the appropriate label. *This can be done automatically by a setting
+     on the project board column.*
```

Comment:
from @bernhold 

Not selectively. We only want curated content ideas to go onto this board. The current automation capabilities only place ALL new issues on a board.

Reply:
I agree that we should only have curated conted. _